rllib/policy/eager_tf_policy.py

```diff
@@ -457,11 +458,51 @@ def loss_initialized(self):
 
         @override(Policy)
         def export_model(self, export_dir):
-            pass
+            """
```

Comment:
Looks good. Could we add a test entry to `tests/test_checkpoint_restore.py` for eager export as well?

Reply:
OK, I will add some code to `tests/test_checkpoint_restore.py`

Reply:
+1